src/lib/url.js

```diff
@@ -0,0 +1,64 @@
+const queryParams = ['include', 'fields', 'sort'];
+
+export const parseInclude = include => {
+  return include ? include.split(',') : [];
```

Comment:
Maybe this could always be a `Set`?

Reply:
Do you think there's an advantage? It will never change, no need to check for unique properties.

Reply:
I thought that we were converting it to set in multiple places but I only found `utils.js::checkIncludesPath`. It'd make that a teeny bit simpler. Not worth blocking this merge on.

Reply:
I had just pulled `queryParams` out of the `compileJsonApiUrl` method, it had just been a hardcoded array of those values before.
Its only use is 
```const queryString = queryParams.filter( ... ).map( ... ).join( ... )```